components/developer/byacc/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2025 David Stes
+# IPS Makefile for Berkeley Yacc
+# Berkeley Yacc was written by Robert Corbett according to the README 
+# Also see ACKNOWLEDGEMENTS and AUTHORS in the sources
+# See sources yysccsid[] = \"@(#)yaccpar	1.9 (Berkeley) 02/21/93\" */"
+# The invisible-island.net byacc version is a version of byacc maintained by Thomas E. Dickey
+# The component version is taken from VERSION in that version of byacc
+# Other versions of byacc exist, see for example https://ftp.math.utah.edu/pub/byacc
+# or see for example the FreeBSD/NetBSD/OpenBSD source code or FreeBSD ports collection
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		byacc
+COMPONENT_VERSION=	20241231
+COMPONENT_SUMMARY=	byacc - Berkeley Yacc
+COMPONENT_DESCRIPTION=  Berkeley Yacc is an LALR(1) parser generator
```

Comment:
This line seems to be unused.

Reply:
It would probably best to remove COMPONENT_DESCRIPTION and move its text over to COMPONENT_SUMMARY .